Content.Client/GameObjects/Components/Atmos/GasAnalyzerVisualizer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Content.Shared.GameObjects.Components.Atmos;
+using Robust.Client.GameObjects;
+using Robust.Client.Interfaces.GameObjects.Components;
+
+namespace Content.Client.GameObjects.Components.Atmos
+{
+    class GasAnalyzerVisualizer : AppearanceVisualizer
+    {
+        public override void OnChangeData(AppearanceComponent component)
+        {
+            base.OnChangeData(component);
+
+            if (component.Deleted)
+            {
+                return;
+            }
+
+            if (!component.Owner.TryGetComponent(out ISpriteComponent sprite))
+            {
+                return;
+            }
+
+            if (component.TryGetData(GasAnalyzerVisuals.VisualState, out GasAnalyzerVisualState visualState))
+            {
+                switch (visualState)
+                {
+                    case GasAnalyzerVisualState.Off:
+                        sprite.LayerSetState(0, "icon");
+                        break;
+                    case GasAnalyzerVisualState.Working:
+                        sprite.LayerSetState(0, "working");
```

Comment:
These two should be read from YAML like other visualizers do.
https://github.com/space-wizards/space-station-14/blob/c00a08f504589c035ea874726bd0a9d16e4ad952/Content.Client/GameObjects/Components/Disposal/DisposalUnitVisualizer.cs#L112-L126